codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AddHttp2Dependency.java

```diff
@@ -38,28 +40,27 @@ public Map<String, Consumer<TypeScriptWriter>> getRuntimeConfigWriters(
             TypeScriptSettings settings,
             Model model,
             SymbolProvider symbolProvider,
-            LanguageTarget target
-    ) {
-        if (!isHttp2Applicable(settings.getService(model))) {
```

Comment:
nit: retain early return?

It improves readability by having a smaller if body and skipping else.